keyboards/ut472/keymaps/Annihilator6000/readme.md

```diff
@@ -0,0 +1,15 @@
+# UT47.2 Planck-style layout switching
```

Comment:
If you could please tweak your keymap directory name - generally, keyboard and keymaps should be lowercase to avoid potential issues with case-insensitive filesystems :)